src/org/omegat/core/statistics/StatsResult.java

```diff
@@ -42,19 +42,27 @@
  */
 public class StatsResult {
     public static final String[] HT_HEADERS = { "", OStrings.getString("CT_STATS_Segments"),
-            OStrings.getString("CT_STATS_Words"), OStrings.getString("CT_STATS_Characters_NOSP"),
-            OStrings.getString("CT_STATS_Characters"), OStrings.getString("CT_STATS_Files") };
+            OStrings.getString("CT_STATS_Words"),
+            OStrings.getString("CT_STATS_Characters_NOSP"),
+            OStrings.getString("CT_STATS_Characters"),
+            OStrings.getString("CT_STATS_Files") };
 
     private static final String[] HT_ROWS = { OStrings.getString("CT_STATS_Total"),
-            OStrings.getString("CT_STATS_Remaining"), OStrings.getString("CT_STATS_Unique"),
+            OStrings.getString("CT_STATS_Remaining"),
+            OStrings.getString("CT_STATS_Unique"),
```

Comment:
These seem to be formatting-only changes. Their presence obscures the semantic changes, so in general I recommend that you not include these in the same PR (or at least put them in their own commit that is labeled appropriately).

Reply:
Indeed. For some reason I forgot to remove them from the commit. Sorry.

Reply:
I've reverted the display modifications and recommitted only the formatting modifications